src/EditorFeatures/Core/Shared/Extensions/IWpfTextViewExtensions.cs

```diff
@@ -1,16 +1,20 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Windows.Media;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.CodeAnalysis.Editor.Shared.Extensions
 {
     internal static class IWpfTextViewExtensions
     {
-        public static void SizeToFit(this IWpfTextView view)
+        public static void SizeToContentsAndScale(this IWpfTextView view, double scaleFactor = 0.75)
         {
             // Computing the height of something is easy.
             view.VisualElement.Height = view.LineHeight * view.TextBuffer.CurrentSnapshot.LineCount;
+            
+            // view.ZoomLevel can cause clipping so use a ScaleTransform instead.
+            view.VisualElement.LayoutTransform = new ScaleTransform(scaleFactor, scaleFactor);
```

Comment:
:arrow_right: The `IWpfTextView.ZoomLevel` property eventually gets used to calculate the height of lines through the old [`IVsTextView.GetLineHeight`](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.textmanager.interop.ivstextview.getlineheight.aspx) API, which returns an integer height for each line. Users who have a font size in the editor that is not divisible by 4 will see pixels trimmed from the bottom of this view. Set your font size to 12 for example and the bug does not appear.